FastSimulation/Calorimetry/python/Calorimetry_cff.py

```diff
@@ -7,8 +7,19 @@
 from FastSimulation.Calorimetry.HcalResponse_cfi import *
 from FastSimulation.Calorimetry.HSParameters_cfi import *
 #from FastSimulation.Configuration.CommonInputs_cff import *
+
+ECALScaleBlock = cms.PSet(
+    ECALResponceScalingParameters = cms.PSet(
+        doScalig = cms.bool( False ),
```

Comment:
Rename ECALResponseScalingParameters to ECALResponseScaling

no need for the doScaling variable,
just scale if the parameter ECALResponseScaling is available